front/src/applications/operationalStudies/hooks/useLazyLoadTrains.ts

```diff
@@ -52,7 +52,7 @@ const useLazyLoadTrains = ({
   const [postTrainScheduleSimulationSummary] =
     osrdEditoastApi.endpoints.postTrainScheduleSimulationSummary.useLazyQuery();
 
-  const { data: { results: rollingStocks } = { results: [] } } =
+  const { data: { results: rollingStocks } = { results: null } } =
```

Comment:
You prefer to default with `null` to oblige us to deal with this case or it's for another reason ?

Reply:
The `null` default is to figure out whether or not the rolling stocks have been loaded. `null` means not yet fetched from the server, an array means the list has been fetched.